src/main/java/com/example/onlinebookshop/entities/Order.java

```diff
@@ -0,0 +1,65 @@
+package com.example.onlinebookshop.entities;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.HashSet;
+import java.util.Set;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.SQLRestriction;
+
+@Getter
+@Setter
+@Entity
+@SQLDelete(sql = "UPDATE orders SET is_deleted = true WHERE id = ?")
+@Table(name = "orders")
+@SQLRestriction(value = "is_deleted=false")
```

Comment:
```suggestion
@SQLRestriction(value = "is_deleted = FALSE")
```
uppercase SQL keywords